lib/osf-components/addon/components/side-nav/x-link/styles.scss

```diff
@@ -1,13 +1,13 @@
 $sidenav-link-height: 40px;
 
-.Link {
+.Link.Link {
```

Comment:
Double down on specificity to override bs button/link styles.

Reply:
Hmm, so bootstrap styles are included *after* osf-components -- maybe we should move our [third-party style imports](https://github.com/CenterForOpenScience/ember-osf-web/blob/a924190812c4f6811a2235810dd0175995e68231/app/styles/headers.scss#L5-L7) from `app/styles/headers` to `osf-components/styles/headers`? Do you think that would remove the need for this specificity-hacking? (sorry for such back-and-forth)

Reply:
This will require a deeper dive into styles bundling and ordering to come up with a good solution.